src/quiz_page_elements/Answer.js

```diff
@@ -0,0 +1,13 @@
+import './answer.css';
+
+export default class Answer {
```

Comment:
This component and `Question.js` could shortened by extending `HTmlElement` like this:
```
import HtmlElement from '../components/htmlElement/HtmlElement';
import './answer.css';

export default class Answer extends HtmlElement {
  constructor(label) {
    super(['quiz-answer'], 'div', label);
  }
}
```